.github/workflows/ci.yml

```diff
@@ -19,28 +19,15 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # pin@v1
```

Comment:
you should still use `rustup` to ensure you have the latest Rust, as the github runners might have a random version pre-installed